frontend/src/modules/Folders/views/FolderView.js

```diff
@@ -30,14 +30,14 @@ import {
 } from 'design';
 import { SET_ERROR, useDispatch } from 'globalErrors';
 import {
+  useClient,
   deleteDatasetStorageLocation,
-  getDatasetAssumeRoleUrl,
-  getDatasetStorageLocation,
-  useClient
+  getDatasetAssumeRoleUrl
 } from 'services';
+import { getDatasetStorageLocation } from '../services';
 
 import { FeedComments } from 'modules/Shared';
-import FolderOverview from './FolderOverview';
+import { FolderOverview } from '../components/FolderOverview';
```

Comment:
Maybe change this to `import { FolderOverview } from '../components'` for consistency, everything else looks good.